bin/oneoff/deprecate_unused_poste_urls.rb

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env ruby
+
+require 'sequel'
+require File.expand_path('../../../pegasus/src/env', __FILE__)
+require src_dir 'database'
+require_relative './poste_urls_constants'
+
+# mark all poste_urls that are not in urls_to_keep as deleted_at = now
+# this will take a long time to run as it updates the entire poste_urls table
+puts "Starting update..."
+
+DB[:poste_urls].update(deleted_at: DateTime.now)
+
+puts "Marked all urls as deleted..."
+
+DB[:poste_urls].where(id: POSTE_URLS_TO_KEEP).update(deleted_at: nil)
```

Comment:
Do we have a way to test this? I think Sequel will generate a long `WHERE` condition (`id IN [x,y,z,d,.........]`) which might generate a SQL statement that’s excessive in length and which might be inefficient to carry out the multi-row update. We might have to iterate over all of the URLs to that we want to keep and update each one individually.

Reply:
I tested locally (using the same list of ids) and it seemed to work fine, although it was against a smaller table. I can update to iterative if it's safer.

Reply:
As long as the list of IDs was the same, we’re good. We’re not writing to this table anymore, so even if the UPDATE of a long list of rows selected by ID is slow, we’re fine.